sdk/src/lib.rs

```diff
@@ -483,6 +483,11 @@ impl PortalSdk {
         Ok(descriptor)
     }
 
+    pub async fn show_mnemonic(&self) -> Result<(), SdkError> {
+        send_with_retry!(self.requests, Request::ShowMnemonic, Ok(Reply::Mnemonic) => break Ok(()))?;
```

Comment:
I think you can probably reuse `Reply::Ok` if you don't need to send anything back in the response